src/window.rs

```diff
@@ -682,6 +682,23 @@ impl Window {
     pub fn set_ime_position<P: Into<Position>>(&self, position: P) {
         self.window.set_ime_position(position.into())
     }
+
+    /// Requests user attention to the window, this has no effect if the application
+    /// is already focused. Providing `None` will unset the request for user attention.
```

Comment:
What does 'user attention' mean? It's probably useful to explain how this might manifest for the user to explain to downstream when they can use it.

Reply:
Added a reference to `UserAttentionType` as that explains it quite well. Is that ok?